pkg/cmd/server/api/util.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+// IsBuildDisabled returns true if the builder feature is disabled.
+func IsBuildDisabled(config *MasterConfig) bool {
+	return IsFeatureDisabled(config, FeatureBuilder)
+}
+
+// IsFeatureDisabled returns true if the feature is listed in the disabled features config
+// of the master config.
+func IsFeatureDisabled(config *MasterConfig, feature string) bool {
```

Comment:
not sure this buys us much... I'd probably leave it out until we have a compelling reason for it